rasa/core/channels/rasa_chat.py

```diff
@@ -71,6 +73,12 @@ async def _fetch_public_key(self) -> None:
                     )
 
     def _decode_jwt(self, bearer_token: Text) -> Dict:
+        if self.jwt_key is None:
+            raise TypeError(
```

Comment:
Is this really a `TypeError` in this case? It rather points to a wrong configuration or some problems with Rasa X to me.

Reply:
The way I understand the code: if this attribute is `None`, it's because [these lines](https://github.com/RasaHQ/rasa/blob/f89b953866d0c85a1228fca6f7852329df5cf522/rasa/core/channels/rasa_chat.py#L90-L91) failed to do their job. And actually we're all ready logging an error [here](https://github.com/RasaHQ/rasa/blob/f89b953866d0c85a1228fca6f7852329df5cf522/rasa/core/channels/rasa_chat.py#L67-L73). So maybe you're right, and this is more a problem with how the user uses rasa/rasa X, and in that case we could raise a RasaException (which wouldn't end up in Sentry)

Reply:
True, but e.g. what if the `key` is a blank string? I'd use some `RasaException` type 👍🏻

Reply:
👍🏻